SU-HSDO/suhumsci
# Arch 123 people nodes

# READY FOR REVIEW

# Summary
- Person content type
- research area content type (for use in person and maybe others)
- publications view fixes

# Needed By (Date)
- soon

# Urgency
- low

# Steps to Test

1. make sure to composer install/update
2. sync to local `blt drupal:sync --site=archaeology --sync-files`
3. ensure the person & research area content types exist
4. use devel generate or manually generate content for research areas, then people, then create some publications and for the author, choose an existing author from the select list
5. view that author's page you choose in step 4
6. ensure the publication you create appears at the bottom and that the layout looks similar to the [mock ups](https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/302442150)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
